frontend/app/templates/components/encryptable/row.hbs

```diff
@@ -35,51 +35,68 @@
+            {{/unless}}
+          </button>
         </div>
-      </div>
-      <div class="col px-0 d-none d-md-block">
-        <button class="btn btn-light copy-btn" {{on "click" this.copyUsername}}
-          disabled={{@encryptable.isUsernameBlank}} >
-          <img class="icon-clippy img d-inline" src="/assets/images/clipboard.svg" alt="clip">
-          {{#unless @encryptable.isUsernameBlank}}
-            <BsTooltip @title={{t "encryptable/credentials.show.copy_username"}} @delayShow="500" />
-          {{/unless}}
-        </button>
-      </div>
 
-      <div class="col-2 pr-0 d-none d-md-block">
-        <div class="hide-wrapper">
-          <Input class="d-inline form-control" @disabled="true" @value={{format-cleartext @encryptable "password"}}/>
-          <a role="button" class="show-text show-text-sm bg-light-blue no-wrap overflow-hidden d-flex align-items-center text-muted {{if this.isPasswordVisible "visibility-hidden"}}"{{on "click" this.showPassword}}>{{t "encryptable/credentials.show.show_password"}}</a>
+        <div class="col-2 pr-0 d-none d-md-block">
+          <div class="hide-wrapper">
+            <Input class="d-inline form-control" @disabled="true" @value={{format-cleartext @encryptable "password"}}/>
+            <a role="button" class="show-text show-text-sm bg-light-blue no-wrap overflow-hidden d-flex align-items-center text-muted {{if this.isPasswordVisible "visibility-hidden"}}"{{on "click" this.showPassword}}>{{t "encryptable/credentials.show.show_password"}}</a>
+          </div>
         </div>
-      </div>
-      <div class="col pl-0 d-none d-md-block">
-        <button class="btn btn-light copy-btn" {{on "click" this.copyPassword}}
-          disabled={{@encryptable.isPasswordBlank}} >
-          <img class="icon-clippy img d-inline" src="/assets/images/clipboard.svg" alt="clip">
-          {{#unless @encryptable.isPasswordBlank}}
-            <BsTooltip @title={{t "encryptable/credentials.show.copy_password"}} @delayShow="500" />
-          {{/unless}}
-        </button>
-      </div>
+        <div class="col pl-0 d-none d-md-block">
+          <button class="btn btn-light copy-btn" {{on "click" this.copyPassword}}
+            disabled={{@encryptable.isPasswordBlank}} >
+            <img class="icon-clippy img d-inline" src="/assets/images/clipboard.svg" alt="clip">
+            {{#unless @encryptable.isPasswordBlank}}
+              <BsTooltip @title={{t "encryptable/credentials.show.copy_password"}} @delayShow="500" />
+            {{/unless}}
+          </button>
+        </div>
+      {{/if}}
 
       <div class="col-md-auto d-none d-md-block">
-        <a class="mx-1" role="button" {{on "click" this.toggleEncryptableEdit}}>
-          <img class="icon-medium-button d-inline" src="/assets/images/edit.svg" alt="edit">
-          <BsTooltip @title={{t "encryptable/credentials.edit.title"}} @delayShow="1000" />
-        </a>
+        {{#unless this.isTransferredCredentials }}
```

Comment:
Remove edit button in encryptable row, because the receive-recrypt only starts, when user clicks in the credentials. If he clicks edit before the receive-recrypt, all fields are null.